R-package/_pkgdown.yml

```diff
@@ -0,0 +1,88 @@
+template:
+  params:
+    bootswatch: cerulean
+
+site:
+  root: ''
+  title: LightGBM, Light Gradient Boosting Machine
+
+authors:
+  Guolin Ke:
+    href: https://github.com/guolinke
+    html: <img src="https://avatars0.githubusercontent.com/u/16040950?s=400&v=4" height="48" /> Guolin Ke
+  Damien Soukhavong:
+    href: https://github.com/Laurae2
+    html: <img src="https://avatars1.githubusercontent.com/u/9083669?s=460&v=4" height="48" /> Damien Soukhavong
+  Yachen Yan:
+    href: https://github.com/yanyachen
+    html: <img src="https://avatars1.githubusercontent.com/u/6893682?s=460&v=4" height="48" /> Yachen Yan
+  James Lamb:
+    href: https://github.com/jameslamb
+    html: <img src="https://avatars1.githubusercontent.com/u/7608904?s=400&v=4" height="48" /> James Lamb
```

Comment:
@jameslamb Oh, no! It's not me, it's @BruceZhaoR!

https://github.com/BruceZhaoR/lightgbm_r/blob/master/pkgdown/_pkgdown.yml

All my work here is based on his repo. So huge thanks @BruceZhaoR! Really appreciate your work, you helped me a lot!

Reply:
@jameslamb We had to hardcode them otherwise no avatar...: https://github.com/Laurae2/LightGBM/blob/d3b5d066ab20c525636037cd30759b6298390d5a/R-package/pkgdown/_pkgdown.yml

We put them big so they stand out better instead of being ridiculously small (default).

Note that it seems the authors don't show up correctly. Only Guolin Ke shows as "Developer".

Reply:
@Laurae2 
> Only Guolin Ke shows as "Developer".

Yeah! I spotted this too!
However, `authors.html` seems to be correctly generated:
https://lightgbm.readthedocs.io/en/docs/R/authors.html

Should I fix something?..